lib/webdriver.py

```diff
@@ -20,7 +20,7 @@
 BASE_URL = "https://mobile.southwest.com"
 LOGIN_URL = BASE_URL + "/api/security/v4/security/token"
 TRIPS_URL = BASE_URL + "/api/mobile-misc/v1/mobile-misc/page/upcoming-trips"
-HEADERS_URL = BASE_URL + "/api/chase/v2/chase/offers"
+HEADERS_URL = BASE_URL + "/api/mobile-air-booking/v1/mobile-air-booking/feature/shopping-details"
```

Comment:
Did southwest change its endpoint on their end?

Reply:
I’m not sure as I didn’t look much into it, but sometimes Southwest will change the requests that are sent when you load the homepage. That’s my best guess as to what happened here. The old URL isn’t being loaded with the homepage, but rather this URL now. 

A better fix to this could potentially be to look for the headers in the request instead of checking for a specific URL.